### PR TITLE
remove obsolete packaging/publishing

### LINKED ISSUE #261: Issues building library package for CentOS/RHEL 8
My first attempt at building a library package for ziti sdk shared library fails around libsodium.
Since the distribution has already libsodium, I have installed `libsodium-devel` and essentially running `cmake` with `-DHAVE_LIBSODIUM=ON` however that fails with:

> Unknown arguments specified

Whether I pass it as a cmd line option to make or change the option to default to `ON` in CMakeLists.txt, the same thing happens.

### PR DESCRIPTION
**Pull request by ekoby**
fix: allow to build from source tarball [fixes #261]

### CODE REVIEW

File: programs/ziti-prox-c/CMakeLists.txt [hunk 1]
```diff
@@ -10,7 +9,8 @@ endif()
 target_link_libraries(ziti-prox-c PUBLIC ziti subcommand)
 target_include_directories(ziti-prox-c PRIVATE ${ziti-sdk_SOURCE_DIR}/inc_internal)
 
-set(ZITI_ARCHIVE_NAME "ziti-prox-c-${ZITI_VERSION}-${CPACK_SYSTEM_NAME}.${archive_sfx}")
+set(ZITI_ARCHIVE_NAME "ziti-prox-c-${PROJECT_VERSION}-${CPACK_SYSTEM_NAME}.${archive_sfx}")
+message(${ZITI_ARCHIVE_NAME})
```
Comment (dovholuknf) [inline programs/ziti-prox-c/CMakeLists.txt:13]: leftover message? or maybe make the message look nicer?

Comment (ekoby) [reply programs/ziti-prox-c/CMakeLists.txt:13]: leftover debug

**Review by dovholuknf — APPROVED** [review]
lgtm

File: programs/ziti-prox-c/CMakeLists.txt (hunk 1 above)
Comment (ekoby) [reply programs/ziti-prox-c/CMakeLists.txt]: removed

**Pull request merged**